models/demos/falcon7b/tests/run_falcon_end_to_end.py

```diff
@@ -91,12 +91,12 @@ class DeviceSetup(Enum):
         "BFLOAT16-L1": {128: (0.65, 0.77, 0.79), 1024: (0.59, 0.85, 0.85), 2047: (0.55, 0.95, 0.94)},
     },
     DeviceSetup.WORMHOLE_B0: {
-        "BFLOAT16-DRAM": {128: (0.89, 0.92, 0.92), 1024: (0.91, 0.94, 0.96), 2047: (0.96, 0.99, 0.98)},
-        "BFLOAT16-L1": {128: (0.89, 0.92, 0.92), 1024: (0.91, 0.94, 0.96), 2047: (0.96, 0.99, 0.98)},
-        "BFLOAT16-L1_SHARDED": {128: (0.89, 0.92, 0.92), 1024: (0.94, 0.95, 0.97), 2047: (0.92, 0.93, 0.95)},
+        "BFLOAT16-DRAM": {128: (0.89, 0.92, 0.91), 1024: (0.92, 0.94, 0.95), 2047: (0.95, 0.96, 0.97)},
```

Comment:
Sorry, I am late to the party.

How is the new matmul "mathematically" different than the old one such that we are getting different results?

Reply:
I think the main difference is that the matmul program config parameters (per core M/N, out subblock h/w) are decided in a slightly different way, leading to a different order of operations, leading to different error accumulation sequences.